controllers/helpers.go

```diff
@@ -301,41 +304,120 @@ func GetSubnetUUID(ctx context.Context, client *prismclientv3.Client, peUUID str
 	return foundSubnetUUID, nil
 }
 
-// GetImage returns an image. If no UUID is provided, returns the unique image with the name.
+// GetImage returns an image UUID. If no UUID is provided, returns the unique image with the name.
```

Comment:
This function retursn an image, not a UUID.

Reply:
fixed the comment